lib/voyage/templates/api_users_controller.rb

```diff
@@ -0,0 +1,56 @@
+module Api
+  module V1
+    class UsersController < BaseApiController
```

Comment:
This is the basic users API endpoint as begun from hivecast and Hypremium. Hopefully this will be helpful, but generic enough to not collide with app-specific requirements.